src/Module/Admin/Storage.php

```diff
@@ -129,6 +129,7 @@ protected function content(array $request = []): string
 				'prefix' => $storage_form_prefix,
 				'form'   => $storage_form,
 				'active' => $current_storage_backend instanceof ICanWriteToStorage && $name === $current_storage_backend::getName(),
+				'set_disabled' => DI::config()->isSetDisabled('storage', 'name'),
```

Comment:
Please add a conditional help text or title attribute to explain why the value is disabled.

Reply:
I add a help text at the top of the setting, so I don't have to repeat myself for every storage